docs/_advanced/executing_workflow_steps.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Executing workflow steps
+lang: en
+slug: executing-steps
+order: 11
+beta: true
+---
+
+<div class="section-content">
+⚠️ Workflow [steps from apps](https://api.slack.com/workflows/steps) is a beta feature. As the feature is developed, **Bolt for JavaScript's API will change to add better native support.**
+
+When your workflow is executed by an end user, your app will receive a `workflow_step_execute` event. This event includes the user's `inputs` and a unique workflow execution ID. Your app must call [`workflows.stepCompleted`](https://api.slack.com/methods/workflows.stepCompleted) with the `outputs` you specified in `workflows.updateStep`.
+</div>
+
+```javascript
+app.event('workflow_step_execute', async ({ event, client }) => {
+  // Unique workflow edit ID
+  let workflowExecuteId = event.workflow_step.workflow_step_execute_id;
+  let inputs = event.workflow_step.inputs;
+
+  client.workflows.stepCompleted({
```

Comment:
should we include an example for `workflows.stepFailed` here?

Reply:
Good point! It's definitely worth calling out, but do you think it'd be fine to leave the code example simple and just note `workflows.stepFailed` in the documentation?

Reply:
That sounds good to me